_pages/about-peer-review.md

```diff
@@ -80,36 +155,49 @@ select them here.
 </div>
 
 <br clear="both">
-
+</div>
 <div class="wide__p_text" markdown="1">
 
-## Why Peer Review for Python Scientific Software?
 
-Peer review of Python scientific software addresses several challenges in the
-open source community:
 
-1. **Citations Support Credit & Acknowledgement:** Peer review provides credit and acknowledgement for the hard work invested in creating and supporting core tools that support open science workflows.
-2. **Standard Packaging & Documentation:** Currently there are numerous ways to create a Python package. Peer review enforces standards thus promoting standardization of Python packaging across the scientific Python ecosystem.
-3. **Removed Redundancy:** Peer review addresses the issue of multiple tools on pyPi that perform similar tasks but have varying levels of documentation, support and maintenance. One maintained package that is broadly useful and maintained will help scientists select tools that they can rely on.
-4. **Long Term Maintenance:** Package maintainers often use their free time to work on Python tools. Once a package is reviewed by pyOpenSci we will keep in touch with the maintainer to ensure continued maintenance of the tool. If a maintainer needs to step down, we will try to find someone new to take over, or sunset the tool as needed.
-5. **Community:** At the core of peer review is community around the tools that scientists need to process their data.
 
-### Who can submit a package?
+
+<!-- ### Who can submit a package?
```

Comment:
Unless you are going to use this in the next week or so, I would remove all comments as they clutter the content and can always be resurrected from the repo history.

Reply:
great point - i'll do that now!!